### PR TITLE
Visitor can view a collection of articles 

### PR DESCRIPTION
**Pull request by gsim1903**
Cypress test for article collection inc stub api with JSON fixture
Article fetch feature
Article card draft

### CODE REVIEW

**Comment by gsim1903** [comment]
Screenshot 2022-03-28 at 08 24 51

File: cypress/integration/placeholderTest.feature.js [hunk 1]
```diff
@@ -6,4 +6,5 @@ describe('Placeholder test to set up CI', () => {
   it('is expected to visit a site', () => {
     cy.get('[data-cy=ci-test]').should('contain.text', 'App')
   });
-});
\ No newline at end of file
+});
+
```
Comment (maxarvid) [inline cypress/integration/placeholderTest.feature.js]: I suggest we delete this test now, it is no longer needed.

File: cypress/integration/userCanSeeCollectioOfArticle.feature.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+describe('user can see a collectino of articles', () => {
```
Comment (maxarvid) [inline cypress/integration/userCanSeeCollectioOfArticle.feature.js]: typo

File: cypress/integration/userCanSeeCollectioOfArticle.feature.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  })
+
+  it('is expected to display a collection of 3 articles', () => {
+    cy.get('[data-cy=articles-list]').children().should('have.length', 3)
+  })
+
+  it('is expected that the first title will be Deep Work', () => {
+    cy.get('[data-cy=articles-list]')
+      .first()
+      .should('have.contain', 'Deep Work')
```
Comment (maxarvid) [inline cypress/integration/userCanSeeCollectioOfArticle.feature.js]: `have.contain`, pretty sure that is not a matcher. Try `contains.text` ?

File: cypress/integration/userCanSeeCollectioOfArticle.feature.js [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+  })
+
+  it('is expected that the first title have a clickable URL', () => {
+    cy.get('[data-cy=articles-list]')
+      .last()
+      .should('have.contain', 'Tipping Point')
+
```
Comment (maxarvid) [inline cypress/integration/userCanSeeCollectioOfArticle.feature.js]: Format code, and this is not testing for clickable URL, it is (maybe) testing for the title of the last article. Consider removing or redrafting.

File: src/App.jsx [hunk 5]
```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import Articles from './Components/Articles.jsx'
+
+
+const App = () => {
+
+  return (
+    <div data-cy="ci-test">App
```
Comment (maxarvid) [inline src/App.jsx]: Ready for deletion now

File: src/Components/Articles.jsx [hunk 6]
```diff
@@ -0,0 +1,33 @@
+import React, { useState, useEffect } from 'react'
+import axios from 'axios'
+// import { Grid } from 'semantic-ui-react'
```
Comment (maxarvid) [inline src/Components/Articles.jsx]: Remove comments before finalizing PR

**Review by maxarvid — CHANGES_REQUESTED** [review]
Very good! I've made a couple of requests but we are almost there.

File: cypress/integration/placeholderTest.feature.js (hunk 1 above)
Comment (gsim1903) [reply cypress/integration/placeholderTest.feature.js]: Removed

File: cypress/integration/userCanSeeCollectioOfArticle.feature.js (hunk 2 above)
Comment (gsim1903) [reply cypress/integration/userCanSeeCollectioOfArticle.feature.js]: Updated

File: cypress/integration/userCanSeeCollectioOfArticle.feature.js (hunk 4 above)
Comment (gsim1903) [reply cypress/integration/userCanSeeCollectioOfArticle.feature.js]: Revised and formatted code. Will include clickable URL test in the next feature.

File: src/App.jsx (hunk 5 above)
Comment (gsim1903) [reply src/App.jsx]: Updated

File: src/Components/Articles.jsx (hunk 6 above)
Comment (gsim1903) [reply src/Components/Articles.jsx]: Removed

**Pull request merged**